src/scripts/containers/MyVideoItemContainer.js

```diff
@@ -0,0 +1,12 @@
+/* @flow */
+
+import { connect } from 'react-redux'
+
+import MyVideoItem from 'components/MyVideoItem'
+import type { RootState } from 'types/ApplicationTypes'
+
+const mapStateToProps = (state: RootState) => ({})
+
+const mapDispatchToProps = () => ({})
+
+export default connect(mapStateToProps, mapDispatchToProps)(MyVideoItem)
```

Comment:
Can this file be deleted? Or will it be needed in the future

Reply:
Let's delete for now.